docs/install.md

```diff
@@ -45,6 +45,11 @@ sudo apt-get update
 sudo apt-get -y install geth
 ```
 
+### Using Parity
```

Comment:
a different spot for this documentation perhaps? this is a doc that elaborates on the installation (bare metal) process. I think the doc that takes one through running is more appropriate (`docs/manual_..._.md` or sth).

Also it's worth to mention how one runs our integration tests with parity (`ETH_NODE=parity mix test --only integration` I presume) - I think we have the test-running invocations documented somewhere, please find and update.

For completeness one can put a quick guide to install parity, in _here_, as is done for geth just above